docs/create-a-neuron.md

```diff
@@ -25,24 +25,51 @@ dfx upgrade
+3. Set the dissolve delay to at least 6 months and confirm the choice.
+
 ## Create a neuron Hotkey
 
-The neuron in the NNS UI (https://nns.ic0.app) needs to be managed with a locally generated private key. This is done though a so called Hotkey.
+The neuron in the NNS UI (https://nns.ic0.app) needs to be managed with a locally generated private key, in order to be able to submit proposals from the command line.
+This is done through a so called Hotkey. In the NNS UI you need to add a locally generated identity as a hotkey, which means that you will be able to perform actions with the NNS UI identity, by using the locally generated identity.
+
 ```bash
 dfx identity new --storage-mode=plaintext neuron-hotkey
+dfx --identity neuron-hotkey identity get-principal
+```
+Example output: `wuyst-x5tpn-g5wri-mp3ps-vjtba-de3xs-w5xgb-crvek-tucbe-o5rqi-mae`
+
+That's it, you now have a local identity that can submit proposals.
+
+## Topping up a neuron
+
+You may need to top up a neuron to send more proposals, either more proposals at the same time, or add balance in case some proposal gets rejected. You need 10 ICP per proposal submission.
+
+The **recommended way** to add more ICPs to the neuron (increase the stake) is through the NNS UI. This should be straightforward and you can just follow the instructions in the NNS UI.
+
+Topping up can also be done from the command line but is slightly more involved. We have built a helper utility for this:
+
+```bash
+dre neuron top-up
 ```
+Will print you the neuron account address. Then you add funds to the printed account address, and after that you need to run:
 
 ```bash
-dfx --identity neuron-hotkey identity get-principal
+dre neuron refresh
 ```
-Example output: `wuyst-x5tpn-g5wri-mp3ps-vjtba-de3xs-w5xgb-crvek-tucbe-o5rqi-mae`
 
-**Note:** This hotkey is used for NNS proposal submissions only.
+So that the staked balance on the neuron is refreshed by the governance canister. Refreshing is typically done automatically when topping up the neuron from the NNS UI.
 
-### Create and Manage Neuron via NNS Frontend Dapp and Internet Identity
+## (Optional and advanced) Hardware wallet identity
+
+In the NNS UI you can also add a hardware wallet as a controlling entity. You will not be able to submit proposals with the hardware wallet entity though. But you can keep all your funds on the hardware wallet.
 
-1. Send at least 11 ICPs to your hardware wallet address.
-2. Navigate to the Neurons tab and create a Neuron by staking at least 10 ICP from your hardware wallet. Confirm the transaction on your hardware wallet.
-3. After the Neuron is created, confirm to "Add NNS Dapp as hotkey" in the dialogue and on your hardware wallet.
-4. Set the dissolve delay to at least 6 months and confirm the choice.
-5. Copy the Neuron ID from the Web UI, for use in the next steps.
+Steps:
+1. After the basic NNS neuron is created, the hardware wallet in the NNS UI.
```

Comment:
Something is missing in the second part of the sentence.

Reply:
Thanks a lot for the detailed review! Much appreciated! :)